.github/workflows/ci.yml

```diff
@@ -712,23 +725,21 @@ jobs:
               ignore exclusion if: >-
                 ${{ needs.build-pt.outputs.have_gcp_secrets == 'true' }}
 
-            - suite: suite-delta-lake-databricks
+            - suite: suite-delta-lake-databricks73
               config: hdp3
-            - suite: suite-delta-lake-databricks
+            - suite: suite-delta-lake-databricks73
+              ignore exclusion if: >-
+                ${{ needs.build-pt.outputs.have_databricks_secrets == 'true' }}
+            - suite: suite-delta-lake-databricks91
+              config: hdp3
+            - suite: suite-delta-lake-databricks91
+              ignore exclusion if: >-
+                ${{ needs.build-pt.outputs.have_databricks_secrets == 'true' }}
+            - suite: suite-delta-lake-databricks104
+              config: hdp3
+            - suite: suite-delta-lake-databricks104
```

Comment:
@findinpath Can you confirm if these suits are running correctly in #14649?

Reply:
Thank you for raising this. @hashhar discovered a regression to be addressed on #14661

Reply:
The second commit is actually a **readability** aspect that I've talked to @hashhar  to move up  to help signaling that `ignore exclusion if` is a label in the matrix and not a github CI keyword.

It is actually not really related to this PR, but was left here because it is a rather small change without any impact on the CI pipeline runtime.

Reply:
> The second commit is actually a **readability** aspect that I've talked to @hashhar to move up to help signaling that `ignore exclusion if` is a label in the matrix and not a github CI keyword/command.

Let's move it to separate.
ci.yml readability aspects tend to be subjective and I'd like to ask original code author for co-review there.